mgmt-hub/docker-compose.yml

```diff
@@ -90,12 +92,14 @@ services:
       - horizonnet
     volumes:
       - ${ETC}/horizon/css.conf:/etc/edge-sync-service/sync.conf.tmpl:${VOLUME_MODE}
-      #todo: why is this needed in this container?
-      - mongovol:/var/edge-sync-service/persist
+      # deploy-mgmt-hub.sh will ensure this dir is empty if we want to use http
+      - ${CERT_DIR}:/home/cssuser/keys:${VOLUME_MODE}
+      #todo: i do not think this needed in this container?
+      #- mongovol:/var/edge-sync-service/persist
     environment:
-      - HZN_EXCHANGE_URL=${HZN_TRANSPORT}://exchange-api:8080/v1
+      - HZN_EXCHANGE_URL=http://exchange-api:8080/v1
```

Comment:
was this intentional to leave as `http`?

Reply:
Yes, please take a look at the note. "Note: This doesn't yet configure the exchange to use https, because the exchange doesn't support that yet (soon)."